### ISSUE TITLE
Is there a way to tell what went wrong?

### ISSUE THREAD

**Issue by camertron** [voidlabs/mosaico#355]
I'm trying to run `Mosaico.init(...)` in my app and it's returning `false` with no additional information. Is there a way to turn on logging or something so I can see what went wrong?

**Comment by Hiswe** [comment]
@camertron mosaico already logs  everything  by default

If you need more logs, you can easily put your own in the  [app.js#init](https://github.com/voidlabs/mosaico/blob/master/src/js/app.js#L151-L173) and recompile mosaico

**Comment by camertron** [comment]
@Hiswe ok, that's good to know. I realized the problem was I didn't have a hash in the URL, and furthermore realized editor.html can't be opened on its own. I opened index.html and noticed that clicking on one of the templates appends a hash and redirects to the editor, which makes sense.

The error that gets printed to the console, i.e. "Missing initialization hash", was confusing to me because of how overloaded the word "hash" is in the programming world. Coming from Ruby, I think of a hash as a data structure of key/value pairs. So, long story short, Mosaico actually _did_ tell me what was wrong, I just didn't understand :)

Thanks for your patience!
